MainModule/Client/Core/Anti.lua

```diff
@@ -48,6 +48,9 @@ return function(Vargs, GetEnv)
 	local Player = service.Players.LocalPlayer
 	local isStudio = select(2, pcall(service.RunService.IsStudio, service.RunService))
 	local Kick = Player.Kick
+	local isXbox = service.GuiService:IsTenFootInterface()
+	local isMobile = service.UserInputService.TouchEnabled and not service.UserInputService.KeyboardEnabled and not service.UserInputService.MouseEnabled
+	local hyperionEnabled = not isXbox and not isMobile and (#tostring(tonumber(string.sub(tostring{}, 8))) > 10)
```

Comment:
So, the solution is to add more hacks to the anti-cheat? Remember, https://github.com/Epix-Incorporated/Adonis/pull/1186, from not too long ago? This is just the opposite of that.

Also, a large majority of genuine Roblox players are on mobile and such (remember Roblox's most popular platform isn't PC), which would still have false detections with this, regardless of it being hacky and easily bypassable.

Reply:
I have nothing to "review" here again per the request. This conversation isn't resolved despite being marked as so, and I have the exact same stance. Nothing in this PR has changed, it's the exact same thing.

Reply:
I made so it only shows in exploitlogs but doesnt disable the AC